addons/project/views/project_views.xml

```diff
@@ -1129,8 +1129,8 @@
                             <field name="partner_id" widget="res_partner_many2one" class="o_task_customer_field"/>
                             <field name="partner_phone" widget="phone" attrs="{'invisible': True}"/>
                             <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}"/>
+                            <field name="recurring_task" attrs="{'invisible': [('allow_recurring_tasks', '=', False), ('active', '=', False)]}"/>
```

Comment:
As per commit guidelines, the commit title should start with a verb. I also corrected small typos.
```txt
[IMP] base,project: improve project and the uninstall wizard
This commit does the following changes:
 - Improve some project filter and views
 - Align the module kanban buttons
 - Sort the module in uninstallation wizard

task-2731708
```